ge/org.osate.ge.ba/src/org/osate/ge/ba/ui/properties/EditEmbeddedTextDialog.java

```diff
@@ -204,12 +225,10 @@ public boolean close() {
 		return super.close();
 	}
 
-	public String getText() {
-		return replacementText;
+	public String getText(final boolean useModifiedSource) {
```

Comment:
Remove this parameter. Doesn't appear to be needed. Everything calls this function with true.

Reply:
I call true when editing the XtextDocument, to replace entire text using doc.set(newSrc).  And I use false when I am just replacing a part of the text in the XtextResource.